include/aws/common/byte_buf.h

```diff
@@ -507,6 +507,8 @@ AWS_STATIC_IMPL struct aws_byte_cursor aws_byte_cursor_from_array(const void *by
  * SPECTRE mitigation purposes.
  */
 AWS_STATIC_IMPL size_t aws_nospec_mask(size_t index, size_t bound) {
+#pragma CPROVER check push
+#pragma CPROVER check disable "unsigned-overflow"
```

Comment:
@bdonlan This pragma disables the unsigned overflow check for this particular code.